packages/taiko-client/driver/preconf_blocks/api.go

```diff
@@ -259,7 +259,7 @@ func (s *PreconfBlockAPIServer) RemovePreconfBlocks(c echo.Context) error {
 		"currentHead", currentHead.Number.Uint64(),
 	)
 
-	if reqBody.NewLastBlockID < canonicalHeadL1Origin.BlockID.Uint64() {
+	if reqBody.NewLastBlockID <= canonicalHeadL1Origin.BlockID.Uint64() {
```

Comment:
I think its OK to let the latest new head to be the highest canonical head?

Reply:
Although this has no effect, it is beyond the scope of preconf.

Reply:
for this API to remove preconf blocks, We SHOULD allow preconfer resets to the latest canonical head, no?

Reply:
Oh I see this way can clean all the preconf blocks.